Makefile

```diff
@@ -460,6 +462,20 @@ build-archive:
 release-unix: clean full build-archive
 	@if [ -f e/Makefile ]; then $(MAKE) -C e release; fi
 
+.PHONY: release-darwin-unsigned
+release-darwin-unsigned: RELEASE:=$(RELEASE)-unsigned
+release-darwin-unsigned: clean full build-archive
+
+.PHONY: release-darwin
+release-darwin: release-darwin-unsigned
+	mkdir -p $(BUILDDIR)/
+	cp ./build.assets/build-common.sh ./build.assets/notarize-binaries.sh $(BUILDDIR)/
+	chmod +x $(BUILDDIR)/notarize-binaries.sh
+	./$(BUILDDIR)/notarize-binaries.sh $(BINARIES)
+	rm $(BUILDDIR)/build-common.sh $(BUILDDIR)/notarize-binaries.sh
+	$(MAKE) build-archive
+	@if [ -f e/Makefile ]; then $(MAKE) -C e release; fi
```

Comment:
So for OSS we have to run `make release-darwin` but for ent we just run `make release`?

Reply:
I reverted most of the drone/dronegen changes that were unnecessary and I believe were causing some confusion. Regardless of whether you're building OSS or ent you should be able to just call `make release`.

As a side note these changes/this PR will only affect OSS tarballs. We don't upload ent/fips tarballs to the download page (https://goteleport.com/download/) so I don't see this as an issue. That being said I can port the changes to ent/fips tarballs if needed.